Storage/rest/advanced_hypermedia.h

```diff
@@ -34,15 +34,21 @@ SOFTWARE.
 #include "hypermedia.h"
 #include "sfinae.h"
 
-CURRENT_STRUCT(AdvancedHypermediaRESTRecordResponse) {
+namespace current {
+namespace storage {
+namespace rest {
+
+CURRENT_STRUCT_T(AdvancedHypermediaRESTRecordResponse) {
+  CURRENT_FIELD(success, Optional<bool>);
```

Comment:
@sompylasar That's almost what we have now, except for the fact that non-advanced Hypermedia returns just an array of URLs for collection. Is it a problem that there is `success: null` field in `data` array entries?